public/app/features/alerting/unified/components/rules/NoRulesCTA.tsx

```diff
@@ -21,7 +21,7 @@ export const NoRulesSplash = () => {
                 </LinkButton>
               )}
               {canCreateCloudRules && (
-                <LinkButton variant="primary" icon="plus" size="lg" href="alerting/new/recording">
+                <LinkButton variant="primary" icon="plus" size="lg" href="alerting/new/grafana-recording">
```

Comment:
this is not correct, as we are checking the `canCreateCloudRules` here. I believe instead of removing the current button, we should add the one for grafana managed recording rules (protected by the `canCreateGrafanaRules`.

Reply:
🤦 Of course, thanks

Reply:
@tomratcliffe can we resolve this outstanding one? Looks good to merge otherwise!